src/passes/Inlining.cpp

```diff
@@ -199,11 +235,55 @@ struct Inlining : public Pass {
     // return whether we did any work
     return inlined.size() > 0;
   }
+
+  bool canInline(int numUses) {
+    return numUses == SINGLE_USE;
+  }
+
+  bool worthInlining(Function* func) {
+    return Measurer::measure(func->body) <= INLINING_SIZE_LIMIT;
```

Comment:
Probably we can increase it, yeah. The reason I'm being conservative in this PR is that the use case we've seen so far involves small functions, and that we do need to optimize after inlining, so a very big function might make that slow (depends on the function we inline into as well, which isn't measured yet...).